x-pack/plugins/observability_ai_assistant/public/service/get_assistant_setup_message.ts

```diff
@@ -40,6 +40,8 @@ export function getAssistantSetupMessage() {
 
   You should autonomously execute these functions - do not wait on the user's permission, but be proactive.
 
+  If you suspect the function you want to execute will change or delete the data in any way, confirm the function with the user first! This is very important.
```

Comment:
I think we probably want to do this in curated UX, but maybe this works fine for now. Have you tried it?